README.md

```diff
@@ -159,7 +159,8 @@ module.exports = {
 
 #### `eslintPath` (default: "eslint")
 
-Path to `eslint` instance that will be used for linting.
+Path to `eslint` instance that will be used for linting.  
+If the `eslintPath` is a folder like a official eslint, or specify a `formatter` option. now you dont have to install `eslint` . 
```

Comment:
Not sure what you are trying to explain here. Can you help me to understand so we can put a better explanation?

Reply:
Sure, when someone specify the `eslintPath`，often means he want use another version of eslint or the eslint under other  `node_modules`.
so he doesn't want to install `eslint` again in current `project/node_modules`